packages/help-center/src/contexts/FeatureFlagContext.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { useContext } from 'react';
+import { FeatureFlags } from '../types';
+
+// Lets typescript know about the feature flags global added through the help-center-script
+declare const helpCenterFeatureFlags: FeatureFlags;
+
+const FeatureFlagContext = React.createContext( {
+	loadNextStepsTutorial: false,
+} );
+
+export const FeatureFlagProvider: React.FC< { children: JSX.Element } > = function ( {
+	children,
+} ) {
+	return (
+		<FeatureFlagContext.Provider value={ helpCenterFeatureFlags }>
+			{ children }
+		</FeatureFlagContext.Provider>
+	);
+};
+
+export function useFeatureFlags() {
+	const featureFlags = useContext( FeatureFlagContext );
+
+	// Trying to call hook outside of component that renders FeatureFlagProvider
+	if ( featureFlags === undefined ) {
```

Comment:
I've added it to the products purchase page (outside of the FeatureFlagProvider) and I'm still able to see the flag value. Can you re-test this and let me know if you're able to replicate it?

Reply:
Good catch! I tested things out again the problem should be fixed in [514e133](https://github.com/Automattic/wp-calypso/pull/65142/commits/514e133047d5b47cbceb1093419800bafe4ce691)

Reply:
Tested and it's working now 👍